src/v2/Apps/Fair/Components/ExhibitorsLetterNav.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { Text, Flex, Swiper } from "@artsy/palette"
+import { Media } from "v2/Utils/Responsive"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { ExhibitorsLetterNav_fair } from "v2/__generated__/ExhibitorsLetterNav_fair.graphql"
+
+const LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("").concat(["0-9"])
+
+interface ExhibitorsLetterNavProps {
+  fair: ExhibitorsLetterNav_fair
+}
+
+export const ExhibitorsLetterNav: React.FC<ExhibitorsLetterNavProps> = ({
+  fair,
+}) => {
+  const letters = fair?.exhibitorsGroupedByName?.map(group => group?.letter)
+
+  const Letters = props => {
+    return (
+      <Flex justifyContent="space-between" {...props}>
+        {LETTERS.map(letter => {
+          const isEnabled = letters?.includes(letter)
+          return (
+            <Letter
+              key={letter}
+              color={isEnabled ? "black100" : "black10"}
+              title={
+                isEnabled ? `View exhibitors starting with “${letter}”` : ""
+              }
+              onClick={() => {
+                if (isEnabled) {
+                  scrollIntoView({
+                    selector: `#jump--letter${letter}`,
+                    offset: 120,
```

Comment:
**question**: Where does this offset come from?

I wonder if we can reuse the offset calculation found here, although I'm not sure what 30 refers to.

https://github.com/artsy/force/blob/f23ebc54255c2de0df547f6443390d00472a833b/src/v2/Apps/BuyerGuarantee/Routes/BuyerGuaranteeIndex.tsx#L69

Reply:
This offset comes from my mind, I just chose the fitting size 😄  Your suggestion looks much better!